[2025-05-16 16:21:11,818 INFO server]: Logging to /lus/eagle/projects/RAPINS/parth/MMA_RadioWave/FederatedFitting/examples/output/result_Server_2025-05-16-16:21:11.txt
Running sum of log likelihood approach
[2025-05-16 16:21:11,826 INFO server]: Running sum of log likelihood approach
Traceback (most recent call last):
  File "/lus/eagle/projects/RAPINS/parth/MMA_RadioWave/FederatedFitting/./examples/octopus/run_server.py", line 108, in <module>
    communicator.publish_server_started_event()
  File "/lus/eagle/projects/RAPINS/parth/MMA_RadioWave/FederatedFitting/src/mma_fedfit/communicator/octopus/octopus_server_communicator.py", line 68, in publish_server_started_event
    self.producer.send(self.topic, value=event)
  File "/home/parthpatel17/.conda/envs/fedfit/lib/python3.10/site-packages/kafka/producer/kafka.py", line 840, in send
    self._wait_on_metadata(topic, timer.timeout_ms)
  File "/home/parthpatel17/.conda/envs/fedfit/lib/python3.10/site-packages/kafka/producer/kafka.py", line 964, in _wait_on_metadata
    raise Errors.KafkaTimeoutError(
kafka.errors.KafkaTimeoutError: KafkaTimeoutError: Failed to update metadata after 59999000.0 secs.
